packages/core/src/mpi/__tests__/match-patient.test.ts

```diff
@@ -0,0 +1,50 @@
+import {
+  jaroWinklerSimilarity,
+  matchingPersonalIdentifiersRule,
+  matchingContactDetailsRule,
+} from "../match-patients";
+import { PatientMPI } from "../shared";
+import { testPatientMPI } from "./test_data";
+
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+
+describe("normalizePatientMPI", () => {
+  it("identifies sampleInclusions as matches", async () => {
+    const searchPatient: PatientMPI = testPatientMPI.sampleSearch[0]!;
+    testPatientMPI.sampleInclusions.forEach((resultData: PatientMPI) => {
+      const resultPatient: PatientMPI = resultData;
+      expect(jaroWinklerSimilarity(searchPatient, resultPatient, 0.96)).toBeTruthy();
+    });
+  });
+
+  it("identifies sampleExclusions as non-matches", async () => {
+    const searchPatient: PatientMPI = testPatientMPI.sampleSearch[0]!;
+    testPatientMPI.sampleExclusions.forEach((resultData: PatientMPI) => {
+      expect(jaroWinklerSimilarity(searchPatient, resultData, 0.96)).toBeFalsy();
+    });
+  });
```

Comment:
tip: create a describe for each function being tested. This way one can run all the tests for that function when changing it, w/o having to test everything else - [example](https://github.com/metriport/metriport/blob/a12981b81602df86643c0723e6de42ec34b0c2c2/packages/core/src/external/commonwell/cq-bridge/__tests__/get-orgs.test.ts#L10).

Reply:
unresolving for visibility, no need to address this now